api/candlepin-api-spec.yaml

```diff
@@ -22,6 +42,53 @@ components:
         requestUuid:
           type: string
 
+    StatusDTO:
+      description: Version and Status information about running Candlepin server
+      discriminator:
+        propertyName: type
+      properties:
+        mode:
+          type: string
+        modeReason:
+          type: string
+        modeChangeTime:
+          type: string
+          format: date-time
```

Comment:
Depending on where we land with the other date things, these may need to become a date format.